### PR TITLE
[Chore] update release workflow to reference schemas in meshery/schemas

### PR DESCRIPTION
**Pull request by MUzairS15**
**Description**
1. When the release workflow for Meshery Cloud runs, the OpenAPI spec in the `meshery/schemas` repository at `openapi/openapi.yml` is referenced.
2. The release version job has been commented out for now, the version in the spec at `meshery/schemas` will be different than the Meshery Cloud version and hence the workflow requires modification which is not done in the PR (in subsequent PR).
3. 
This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/meshery-cloud-release.yml [hunk 1]
```diff
@@ -181,11 +200,11 @@ jobs:
         path: meshery-cloud
         fetch-depth: 1
         token: ${{ secrets.GH_ACCESS_TOKEN }}
-    - name: Build Docs
-      run:  |
-        cd meshery-cloud
-        npm i -g @redocly/cli@latest
-        make docs-build
+    - name: Download artificats (Built API docs)
```
Comment (MUzairS15) [inline .github/workflows/meshery-cloud-release.yml:203]:
```suggestion
    - name: Download artifacts (Built API docs)
```

File: .github/workflows/meshery-cloud-release.yml [hunk 2]
```diff
@@ -118,11 +137,11 @@ jobs:
         repository: layer5io/meshery-cloud
         fetch-depth: 1
         token: ${{ secrets.GH_ACCESS_TOKEN }}
-    - name: Build Docs
-      run:  |
-        cd meshery-cloud
-        npm i -g @redocly/cli@latest
-        make docs-build
+    - name: Download artificats (Built API docs)
```
Comment (MUzairS15) [inline .github/workflows/meshery-cloud-release.yml:140]:
```suggestion
    - name: Download artifacts (Built API docs)
```

**Comment by MUzairS15** [comment]
@theBeginner86 @leecalcote

**Comment by leecalcote** [comment]
@MUzairS15 is this moving the API docs from being published under meshery.layer5.io to meshery/schemas (to no where, since there isn't a website at meshery/schemas)?

Some to be aware of is the fact the Docs comes with a (no-so-usable) redocly integration. See the example on https://docs.layer5.io/cloud/reference/

**Comment by MUzairS15** [comment]
No since the openAPI spec has been moved to schemas repo, so this PR is for using the spec in schemas repo instead of meshery-cloud spec.

**Comment by leecalcote** [comment]
@MUzairS15 @theBeginner86, please see the merge conflict. Have we have you already taken care of this in a different PR? If not, as soon as the emerge conflicts are fixed, let's merge.

**Comment by leecalcote** [comment]
@MUzairS15 @Yashsharma1911 ... now opportune time to revisit this.

**Comment by leecalcote** [comment]
@MUzairS15

**Comment by leecalcote** [comment]
@MUzairS15

**Comment by leecalcote** [comment]
@MUzairS15

**Comment by leecalcote** [comment]
@MUzairS15 I assume that this PR is not needed. Is that correct?

**Comment by leecalcote** [comment]
@MUzairS15, how might I offer help here?

**Comment by MUzairS15** [comment]
Todo: Does provider schema be inside schemas?

**Pull request closed without merging**